fastestimator/estimator.py

```diff
@@ -106,6 +106,16 @@ def network(self) -> BaseNetwork:
     def traces(self) -> List[Union[Trace, Scheduler[Trace]]]:
         return self.system.traces
 
+    @ property
+    def steps_per_epoch(self,mode):
```

Comment:
this property can be inside the system object so that you don't have to pass the mode in order to query it

Reply:
It might cause an issue here right? Will this property be able to work as expected here?
![image](https://user-images.githubusercontent.com/44550569/143147272-019c4024-8f1c-4eac-89f4-d9207c650ed3.png)